app/controllers/UsersController.scala

```diff
@@ -281,7 +302,27 @@ class UsersController @Inject()(messagesApi: MessagesApi,
         Future.successful(BadRequest(views.html.users.updateuser(formWithErrors)))
       },
       userInfo => {
-        usersRepository.update(UpdatedUserInfo(userInfo.email, userInfo.active, userInfo.ban, userInfo.password))
+        usersRepository.update(UpdatedUserInfo(userInfo.email, userInfo.active, userInfo.ban,userInfo.coreMember,userInfo.admin, userInfo.password))
+          .flatMap { result =>
+            if (result.ok) {
+              Logger.info(s"User details successfully updated for $email")
+              Future.successful(Redirect(routes.UsersController.manageUser(1, None))
+                .flashing("message" -> s"Details successfully updated for $email"))
+            } else {
+              Future.successful(InternalServerError("Something went wrong!"))
+            }
+          }
+      })
+  }
+
+  def updateUserByAdmin(): Action[AnyContent] = superUserAction.async { implicit request =>
```

Comment:
Action requires super user permissions, why is method name "updateUserByAdmin" then?